config/config.yaml

```diff
@@ -78,6 +116,6 @@ service:
   extensions: [health_check, pprof, zpages, bearertokenauth/github]
   pipelines:
     metrics:
-      receivers: [gitprovider, prometheus]
+      receivers: [github, gitprovider, prometheus]
```

Comment:
This in general is just an example config, not meant to necessarily be used. We can remove what we feel is necessary, like prometheus, and gitprovider.

Reply:
Roger, I wanted to clear up most of that